rpcserverhelp.go

```diff
@@ -618,6 +629,9 @@ var rpcResultTypes = map[string][]interface{}{
 	"searchrawtransactions": {(*string)(nil), (*[]btcjson.SearchRawTransactionsResult)(nil)},
 	"sendrawtransaction":    {(*string)(nil)},
 	"setgenerate":           nil,
+	"addminingaddr":         nil,
+	"delminingaddr":         nil,
+	"listminingaddrs":       nil,
```

Comment:
This returns a slice of strings, so it should be:

```Go
{(*[]string)(nil)},
```

That would also trigger a test failure due to the missing `listminingaddrs--result0` help key I commented on above.